README.md

```diff
@@ -233,46 +233,47 @@ If you're implementing your authentication logic in a Fragment, you'll need to w
     }
     ```
 
-Explanation of the parameters:
+	Explanation of the parameters:
     
-  * Resource is required and is the resource you are trying to access.
-  * Clientid is required and comes from the AzureAD Portal.
-  * You can setup redirectUri as your packagename. It is not required to be provided for the acquireToken call.
-  * PromptBehavior helps to ask for credentials to skip cache and cookie. 
-  * Callback will be called after authorization code is exchanged for a token. 
+	* Resource is required and is the resource you are trying to access.
+	* Clientid is required and comes from the AzureAD Portal.
+	* You can setup redirectUri as your packagename. It is not required to be provided for the acquireToken call.
+	* PromptBehavior helps to ask for credentials to skip cache and cookie. 
+	* Callback will be called after authorization code is exchanged for a token. 
   
-  The Callback will have an object of AuthenticationResult which has accesstoken, date expired, and idtoken info. 
+	The Callback will have an object of AuthenticationResult which has accesstoken, date expired, and idtoken info. 
+	
+	Optional:  **acquireTokenSilent**
```

Comment:
I do not know what is "optional" about acquireTokenSilent... Usually "Optional" phrase is used for describing optional parameters. In this case, we are talking about a different method. What we should be explaining is about why one should call or not call acquireTokenSilent.